### PR TITLE
Changed alignment under Project section on frontpage

### PR DESCRIPTION
**Pull request by shreyaag770**
changes asked in PR  #237 is done in this PR.

### CODE REVIEW

File: antora-ui-camel/src/css/frontpage.css [hunk 1]
```diff
@@ -260,4 +261,7 @@ section.frontpage h1 {
   section.frontpage.projects .project p:nth-child(3) {
     text-align: center;
   }
-}
+  section.frontpage.projects .project h1 {
+    text-align: center;
+  }
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: This change fails the lint check.

**Comment by shreyaag770** [comment]
Done ! @zregvart, please review it again.

**Comment by zregvart** [comment]
I've rebased and fixed the conflict. We don't have the preview so here's the current and the proposed screenshots.

Current

Screenshot_2020-03-30 Home - Apache Camel(1)

Proposed

Screenshot_2020-03-30 Home - Apache Camel

**Comment by shreyaag770** [comment]
Thanks! @zregvart I have some suggestion for better alignment, what about the right alignment of headings under the project section.

preview:
z1

**Comment by shreyaag770** [comment]
@zregvart Please, have a look at PR #296. I suppose the right alignment of headings looks better.

Preview:

z1

**Comment by zregvart** [comment]
@shreyaag770 its difficult to follow up on your pull requests can you close the ones that don't need to be merged and leave only the ones that you wish to have merged?

**Comment by shreyaag770** [comment]
ok @zregvart and I am really sorry, I am making your work hectic.

**Pull request closed without merging**